docs/data/material/getting-started/learn/learn.md

```diff
@@ -1,61 +1,65 @@
-# Learn MUI
+# Learning Resources
```

Comment:
```suggestion
# Learning resources
```

Reply:
"Learn" on the side nav is kind of vague. Should we change that to "Learning Resources"?

Reply:
"Learning resources" used on both the sidebar and title sounds great to me 👍